earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -30,6 +30,9 @@
 #include "common/SharedString.h"
 #include "StorageManagerAssetHandle.h"
 #include "AssetSerializer.h"
+#include "AssetFactory.h"
```

Comment:
Are these lines necessary? Since you moved Find and ValidateRefForInput, I don't think they're used.